src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Todo List App</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <div class="todo-list">
+      <h3>Today's To Do.</h3>
+      <div class="add-task">
+        <input
+          type="text"
+          name=""
+          placeholder="Add your task here..."
+          class="input-task"
+          required
+        />
+        <i class="fa fa-arrow-left"></i>
+      </div>
+      <div class="display"></div>
+    </div>
+    <script
+      src="https://kit.fontawesome.com/a2d92cea5e.js"
+      crossorigin="anonymous"
+    ></script>
```

Comment:
Please, Switch script to the head tag and add the defer attribute, so that the script runs after everything else 🙃.